include/dsn/dist/meta_state_service.h

```diff
@@ -25,20 +25,46 @@
  */
 # pragma once
 
-# include <dsn/service_api_cpp.h>
+#include <dsn/service_api_cpp.h>
+#include <string>
+#include <functional>
+#include <dsn/internal/task.h>
```

Comment:
files under internal is for src/core use only. applications can use the abstractions in service_api_c and service_api_cpp instead.